src/main/java/org/nest/symboltable/predefined/PredefinedFunctions.java

```diff
@@ -147,13 +147,13 @@
 
     final MethodSymbol i_sum = createFunctionSymbol(CURR_SUM);
     i_sum.addParameterType(getType("pA"));
-    i_sum.addParameterType(getBufferType());
+    i_sum.addParameterType(getRealType());
```

Comment:
I'm quite out of my league here, so it's just a question to try to understand how the inner functions work: I thought you kept the ``BufferType``, so why are the functions now associated with ``RealType``?